opentelemetry-sdk/src/opentelemetry/sdk/context/propagation/b3_format.py

```diff
@@ -100,13 +101,22 @@ def extract(cls, get_from_carrier, carrier):
         )
 
     @classmethod
-    def inject(cls, context, set_in_carrier, carrier):
-        sampled = (trace.TraceOptions.SAMPLED & context.trace_options) != 0
+    def inject(
+        cls, span, set_in_carrier, carrier
+    ):  # pylint: disable=arguments-differ
+        sampled = (
+            trace.TraceOptions.SAMPLED & span.context.trace_options
+        ) != 0
         set_in_carrier(
-            carrier, cls.TRACE_ID_KEY, format_trace_id(context.trace_id)
+            carrier, cls.TRACE_ID_KEY, format_trace_id(span.context.trace_id)
         )
         set_in_carrier(
-            carrier, cls.SPAN_ID_KEY, format_span_id(context.span_id)
+            carrier, cls.SPAN_ID_KEY, format_span_id(span.context.span_id)
+        )
+        set_in_carrier(
+            carrier,
+            cls.PARENT_SPAN_ID_KEY,
+            format_span_id(span.parent.context.span_id),
```

Comment:
the parent can be empty, so there should be a conditional here to check that first.

Reply:
Ok, added the check :+1: